Composer/packages/client/src/components/PluginHost/PluginHost.tsx

```diff
@@ -4,7 +4,7 @@
 /** @jsx jsx */
 import { jsx, SerializedStyles } from '@emotion/core';
 import React, { useState, useEffect, useRef } from 'react';
-import { Shell } from '@bfc/types';
+import { Shell } from '@botframework-composer/types';
```

Comment:
maybe I missed it, but I don't see `@botframework-composer/types` in `dependencies` or `devDependencies` of client

Reply:
Good catch.